src/issuance.rs

```diff
@@ -1327,18 +1323,15 @@ mod tests {
         let prev_finalized = HashSet::new();
 
         // 1. Try too long description
-        signed
-            .actions
-            .first_mut()
-            .modify_descr(String::from_utf8(vec![b'X'; 513]).unwrap());
+        signed.actions.first_mut().modify_descr(vec![b'X'; 513]);
```

Comment:
please fix typos for comment (all for this test)

Reply:
Done